docs/providers/aws/guide/serverless.yml.md

```diff
@@ -31,6 +31,7 @@ provider:
   timeout: 10 # The default is 6
   deploymentBucket: com.serverless.${self:provider.region}.deploys # Deployment bucket name. Default is generated by the framework
   role: arn:aws:iam::XXXXXX:role/role # Overwrite the default IAM role which is used for all functions
+  cfnRole: arn:aws:iam::XXXXXX:role/role # ARN of an IAM role for CloudFormation service. If specified, CloudFormation uses the role's credentials
```

Comment:
Might be worth noting that if excluded, CloudWatch will assume the role previously associated with the stack. If I understand correctly, in this case most cases means it will continue to use the same role.

http://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html